css/style.css

```diff
@@ -0,0 +1,209 @@
+#nav-bar {
+    display: grid;
+    grid-template-columns: 0fr repeat(3, 1fr);
+    height: fit-content;
+}
+
+.nav-items ,.logo{
+    border: 2px solid #25041D;
+    border-left: 1px solid #25041D;
+    border-right: 1px solid #25041D;
+    display: grid;
+    justify-content: center;
+    align-content: center;
+    background-color: #5C0A47;
+    height: 5vh;
+    color: white;
+    font-size: 5vh;
+}
+.nav-items{
+    font-size: 2vh;
+    font-style: italic;
+    cursor: pointer;
+}
+
+
+.clicked-nav-item,.nav-items:hover {
+    border-bottom: 0;
+    background-color: #F4F9E9;
+    color: black;
+}
+
+#content-container {
```

Comment:
try to clean your style.css file. remove empty spaces and any classes/ids you do not use